client/src/components/existing-request/existing-request-search.vue

```diff
@@ -1,33 +1,33 @@
 <template>
-  <v-form v-model="isValid" lazy-validation @submit="handleSubmit">
-  <v-row align="center" class="pa-6 pl-8 pr-9 copy-normal">
-    <v-col cols="12" class="pa-4">
-      Enter the NR Number or Business Name and either the Applicant's Phone or Email</v-col>
-    <v-col cols="12" class="px-4 mt-n4 red--text" v-if="errorMessage">{{ errorMessage }}</v-col>
-    <v-col cols="1" class="max-height">
-      <v-img src="../../assets/images/one-icon.png"
-                  contain
-                  height="60" />
-    </v-col>
-    <!-- NR NUMBER -->
+  <v-form v-model="isValid" lazy-validation @submit="handleSubmit" class="pa-10">
```

Comment:
OK, so, the bulk of the work in here is to untangle the div paddings/margins.
Now, you can click on an element (in dev tools) and you will see that divs do not have extra (or negative) margins. Everything is tight (except where whitespace is added for separation). This is the way it should be :)